checklist.md

```diff
@@ -0,0 +1,53 @@
+## F3 Activation Procedure (FRC-0099)
+
+This document outlines the procedure for activating F3 on the Filecoin network, adhering to FRC-0099. The process involves a multi-step approach, leveraging a smart contract to ensure transparency and community consensus.
+
+1.  **Parameter Selection and Bootstrap Epoch Determination**
+    *   Goal: Establish the "good parameter set" for F3 and the initial epoch for its activation.
+    *   Action: Analyze nv25 passive testing results. Consider network performance, stability, and community input.
```

Comment:
```suggestion
    *   Action: Analyze nv25 passive testing results. Consider network performance, stability, and community input.
    * Status: not started
```

I'm thinking we add a status section to each step that we'll update as we make progress through the checklist